src/ICeWall/src/ICeWall.tcl

```diff
@@ -4432,22 +4525,51 @@ namespace eval ICeWall {
 
   proc init_index {} {
     variable cells
-    variable idx
+    variable cell_idx
 
+    set cell_idx {fill 0}
     foreach key [dict keys $cells] {
-      dict set idx $key 0
+      dict set cell_idx $key 0
+    }
+  }
+
+  variable unfilled_padring_segments {}
+  proc add_unfilled_padring_segment {side pos1 pos2} {
+    variable unfilled_padring_segments
+
+    dict lappend unfilled_padring_segments $side [list [expr min($pos1,$pos2)] [expr max($pos1,$pos2)]]
+  }
+
+  proc clear_unfilled_padring_segments {} {
+    variable unfilled_padring_segments
+
+    set unfilled_padring_segments {}
+  }
+
+  proc report_unfilled_padring_segments {} {
+    variable unfilled_padring_segments
+
+    if {[dict size $unfilled_padring_segments] > 0} {
+      dict for {side gaps} $unfilled_padring_segments {
+        utl::info PAD 253 "Unfilled gaps in the padring on $side side"
+        foreach gap $gaps {
+          utl::info PAD 254 "    [ord::dbu_to_microns [lindex $gap 0]] -> [ord::dbu_to_microns [lindex $gap 1]]"
```

Comment:
There should probably be warnings instead

Reply:
Ok - done